CCM-17047: Add client , campaign and supplier name to Grafana dashboard (#543)
NHSDigital/nhs-notify-supplier-api: README.md

#### Prerequisites and Configuration

> - create the file `~/.aws/config` with the following contents:

Reviewer: ```suggestion
- Create the file `~/.aws/config` with the following contents:
```

  ```dsconfig
  [profile ]
  region = eu-west-2
  output = json

  [profile supplier-dev]
  sso_start_url = https://d-9c67018f89.awsapps.com/start#/
  sso_region = eu-west-2
  sso_account_id = 820178564574
  sso_role_name = nhs-notify-bc-developer
  region = eu-west-2
  output = json

  [profile supplier-nonprod]
  sso_start_url = https://d-9c67018f89.awsapps.com/start#/
  sso_region = eu-west-2
  sso_account_id = 885964308133
  sso_role_name = nhs-notify-bc-developer
  region = eu-west-2
  output = json
  ```

- in your `~/.bashrc` or `~/.zshrc` add the export `export AWS_PROFILE=supplier-dev`, or whichever profile you need

> - In the project's root directory create an `.env` file based on the `.env.template` file and fille variables as needed.

Reviewer: ```suggestion
- In the project's root directory create an `.env` file based on the `.env.template` file and fill variables as needed.
```

> - create the file `~/.npmrc` with the contents:

Reviewer: ```suggestion
- Create the file `~/.npmrc` with the contents:
```

   ```dsconfig
    # Authenticate to GitHub Packages for github.com
    //npm.pkg.github.com/:_authToken=<Insert your Github PAT (Personal Access Token)>

    # Package is scoped under @org, set registry for that scope
    @nhsdigital:registry=https://npm.pkg.github.com
   ```

- Install `node` (to run `npm install` and build the project)

> - install `aws cli` to be able to connect to AWS (needed for some tests)

Reviewer: ```suggestion
- Install `aws cli` to be able to connect to AWS (needed for some tests)
```

- If AWS CLI calls are blocked by a firewall (e.g. Zscaler), you need to add the custom certificates in the location `/scripts/devcontainer/custom-ca-certs`

> - install `docker` or `Rancher` for containerisation

Reviewer: ```suggestion
- Install `docker` or `Rancher` for containerisation
```

>   - Utilised the devcontainer, for pre reqs and configuration.

Reviewer: I think this could be removed, it's not saying anything useful, the new instructions are much better

>   - You should open in a devcontainer or a Github workspaces. (In VSCode -> open control palet -> "Dev containers: rebuild without cache and reopen in container")

Reviewer: ```suggestion
  - You should open in a devcontainer or a Github workspaces. (In VSCode -> Open Command Palette -> "Dev containers: rebuild without cache and reopen in container")
```
